sdk/eventhubs/azure-messaging-eventhubs/migration-guide.md

```diff
@@ -303,22 +308,20 @@ Disposable subscription = consumer.receiveFromPartition("my-partition-id", Event
 // Keep fetching events
 // When you are finished, dispose of the subscription.
 subscription.dispose();
-```
 
-See [`ConsumeEvents.java`][ConsumeEvents] for a sample program demonstrating this.
+// Dispose of any resources.
+consumer.close();
+```
 
-### Migrating code from `EventProcessorHost` to `EventProcessorClient` for receiving events
+### Receiving events from all partitions
```

Comment:
This section should be moved above "Receiving events from single partition" as this is the recommended approach and customers most likely would be using EPH.